tests/unit/Application/Metadata/JsonGeneratorTest.php

```diff
@@ -433,13 +410,13 @@ public function test_it_can_build_oidc_entity_data_for_new_entities()
                             'arp' => 'arp',
                         ),
                     'type' => 'saml20-sp',
+                    'state' => 'testaccpeted',
```

Comment:
typo?

Reply:
In the test this value is arbitrary, but a typo nonetheless.